### DISCUSSION TITLE
LibreHatti to erpNext

### DISCUSSION THREAD

**Discussion by hsrai** [GreatDevelopers/LibreHatti#550] [Q&A]
This office Automation Software was developed by students of GNDEC, mostly during their training at Testing and Consultancy Cell.

Now, we are thinking to upgrade it.

The one idea which we are exploring at the moment is to migrate to [erpNext](https://erpnext.org/).

Also see [erpNext com site](https://erpnext.com/)

What do you say.

**Comment by mandeeps708** [comment]
Do we want to move to our own self-hosted ERP Next instance and customize that according to our needs?

Does that also mean we will eventually stop using LibreHatti at TCC due to the lack of support it needs?

**Reply by hsrai** [reply]
Will be hosted on Server of TCC. If we migrate, then no new work will be done on LibreHatti, but keep it up for some time, to make access previous data.

**Comment by Bipinjot** [comment]
> This office Automation Software was developed by students of GNDEC, mostly during their training at Testing and Consultancy Cell.
> 
> Now, we are thinking to upgrade it.
> 
> The one idea which we are exploring at the moment is to migrate to [erpNext](https://erpnext.org/).
> 
> Also see [erpNext com site](https://erpnext.com/)
> 
> What do you say.

Hi Sir, 
Upgrade as in? Do you mean in terms of the architecture or tech stack?

**Reply by hsrai** [reply]
1. To run it on latest OS.
2. To incorporate more functionality, as per need of client
3. To make it user friendly
4. To make it mobile friendly
5. To resolve issue of maintenance.

Be believe technology is just a vehicle. Any technology which do the job, is OK for us.

**Comment by jasvir99** [comment]
There are various aspects of LH that should be addressed by erpNext. Let me give it a try before saying anything.

**Reply by jasvir99** [reply]
@hsrai I don't believe this should be good enough for TCC. To make it workable by TCC staff, we have to spend time in customizing it which will not be easy. Instead we should spend time in analyzing pain points in LH and fix those. Rest it depends upon the team.

**Comment by Bipinjot** [comment]
Hi All!
Thank you writing to me. That’s great. How exactly can I contribute to the
project?

Best
Bipinjot Kaur Hara

On Mon, 22 Feb 2021 at 7:35 AM, Jasvir Singh Grewal <
notifications@github.com> wrote:

> There are various aspects of LH that should be addressed by erpNext. Let
> me give it a try before saying anything.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GreatDevelopers/LibreHatti/discussions/550#discussioncomment-392208>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGEXR75X4T4CPDI532PMPJTTAG3VNANCNFSM4X6S6A2A>
> .
>
-- 
Greetings of the day!





Regards
Bipinjot Kaur Hara

**Reply by hsrai** [reply]
It all depends on your skill-set and area of interest. We will keep posting what we are doing, and you may extent help or mentorship, wherever you feel to so so.

**Reply by SatinderpalSingh** [reply]
BTW what is your skill-set and to be a part of this project take part in discussions

**Comment by mohitrana67** [comment]
Hello Sir,

I can contribute to customizing and building data charts that can be used for faster analysis.

Moreover, I can help in building data-driven user-friendly dashboards according to user needs, and once we have enough data we can start working on creating an Automated Model for future user requests.

These are just my thoughts, said that Always happy to hear more and will be happy to work on any aspect of the project related to Python, Data Analysis, Databases, and Automation using scripting. 

Hope to be an integral part of this team.

Thanks

**Comment by amrit3701** [comment]
> This office Automation Software was developed by students of GNDEC, mostly during their training at Testing and Consultancy Cell.
> 
> Now, we are thinking to upgrade it.

Here what **updrade** means?

Does it mean there are some issues with LibreHatti and we to need to fix them? Or to develop a new LibreHatti on the top of erpNext system (which is not a good idea as said by @jasvir99).

Maybe one can try which LibreHatti features are not present in the erpNext system.

**Comment by jasvir99** [comment]
The solution is not to switch over to erpNext as it'll require customization, and training also. We have to list down the pain points and start resolving the issues in LH.

**Reply by SatinderpalSingh** [reply]
I know that we have deliver a huge time on Librehatti but it is not built according to the standards of a company or not change ready when needed. We are dealing with different tax slabs after GST. If we will try to redefine the Librehatti, we need experience from the commerce background and it will take a huge time to deliver such system. 

And Librehatti satisfies only some of the requirements of TCC, but it is a part of college. So if we go for ErpNext then we will be able to cover whole college and sister institutes.

**Comment by jasvir99** [comment]
@hsrai can we conclude this discussion? In my opinion, erpNext is not the solution. We have to fix LH.

**Reply by hsrai** [reply]
Expecting response of current team members before I conclude.

BTW, have you ( @jasvir99 ) explored erpNext?

**Reply by jasvir99** [reply]
I have tried the demo instance but it is not what we expected. There is option for purchase order but it need to be customized as per TCC needs. Only 20-25% can be used but all other stuff needs to be customized more or less.

**Reply by SatinderpalSingh** [reply]
Are the needs/work_flow of TCC are different from other companies, if yes then they need to be redefined according to some standard. A good software assists in elimination of human errors, as a result, improves effectiveness and consistency.

**Reply by Sudhanshu-Dubey14** [reply]
Agreeing with @SatinderpalSingh sir we need to see why the work flow/needs of TCC are different from industry standards and by how much. If we don't adapt to industry standards, this issue of upgrading our custom software with every policy change in the outside world (like GST) will persist.
On the other hand, if we put efforts now to adapt to standards we will benefit in the future.

So in my view, we should first involve the TCC staff in this discussion, get their views on possible upgrades in their work flow and then accordingly discuss whether LH should be upgraded or ERPNext should be customised.

**Comment by vigasdeep** [comment]
> This office Automation Software was developed by students of GNDEC, mostly during their training at Testing and Consultancy Cell.
> 
> Now, we are thinking to upgrade it.
> 

do we have a list of problems that we are looking forward to be solved/fixed if and when we upgrade? If not, I’d suggest to compile a list and decide accordingly.

**Reply by SatinderpalSingh** [reply]
Yes we have the list of requirements now, and we have to make system accordingly. What is the best way to compile, i mean how we will decide the flow of work to design the system.

**Reply by Divya-Juhi-Singh** [reply]
@SatinderpalSingh as you commented above,
>And Librehatti satisfies only some of the requirements of TCC, but it is a part of college. So if we go for ErpNext then we will be able to cover whole college and sister institutes.

Apart from this, can we have a list of pin points where ErpNext is expected to lead over Librehatti? 
This comparison can conclude in a better way, as I expect.
Meanwhile, can you also provide the list of requirements that is expected to be in the system?

**Reply by vigasdeep** [reply]
> Yes we have the list of requirements now, and we have to make system accordingly. What is the best way to compile, i mean how we will decide the flow of work to design the system.

Can you post the list here?

**Reply by SatinderpalSingh** [reply]
Actually we are planning to implement the ERP for whole trust plus all the institutes of the trust. So the basis infrastructure of the this will be like this:

NSET
|
GNDEC|
|            |TCC
|            |CSE
|            |IT, etc
|
|GNDPC
|NSPS

Making NSET as the parent company, we can add GNDEC as child company and TCC and other depts. as the childs. and similarly the other schools and colleges of the trust will be treated as the children of the NSET

And we wish to test it with the work flow of TCC.

**Comment by margret45** [comment]
oh it is much interesting

**Comment by margret45** [comment]
I really like [erp software dubai](https://firstbit.ae/) because with its help I was able to stabilize my business. The business improved a lot and the income began to grow rapidly. There were a lot of buyers. I am very satisfied that there is such an assistant in the business who will help me with everything. I have been using it for two years and it has never let me down.
